packages/endpoint/0.12.0/dataset/file/fields/fields.yml

```diff
@@ -0,0 +1,705 @@
+- name: '@timestamp'
+  level: core
+  required: true
+  type: date
+  description: 'Date/time when the event originated.
+
+    This is the date/time extracted from the event, typically representing when
+    the event was generated by the source.
+
+    If the event source has no original timestamp, this value is typically populated
+    by the first time the event was received by the pipeline.
+
+    Required field for all events.'
+  example: '2016-05-23T08:05:34.853Z'
+- name: message
+  level: core
+  type: text
+  description: 'For log events the message field contains the log message, optimized
+    for viewing in a log viewer.
+
+    For structured logs without an original message field, other fields can be concatenated
+    to form a human-readable summary of the event.
+
+    If multiple messages exist, they can be combined into one message.'
+  example: Hello World
+- name: agent
+  title: Agent
+  group: 2
+  description: 'The agent fields contain the data about the software entity, if
+    any, that collects, detects, or observes events on a host, or takes measurements
+    on a host.
+
+    Examples include Beats. Agents may also run on observers. ECS agent.* fields
+    shall be populated with details of the agent running on the host or observer
+    where the event happened or the measurement was taken.'
+  footnote: 'Examples: In the case of Beats for logs, the agent.name is filebeat.
+    For APM, it is the agent running in the app/service. The agent information does
+    not change if data is sent through queuing systems like Kafka, Redis, or processing
+    systems such as Logstash or APM Server.'
+  type: group
+  fields:
+  - name: id
+    level: core
+    type: keyword
+    ignore_above: 1024
+    description: 'Unique identifier of this agent (if one exists).
+
+      Example: For Beats this would be beat.id.'
+    example: 8a4f500d
+  - name: type
+    level: core
+    type: keyword
+    ignore_above: 1024
+    description: 'Type of the agent.
+
+      The agent type stays always the same and should be given by the agent used.
```

Comment:
this has probably always been there and not actually related to your change, but the wordings a bit weird. `always stays the same` would be smoother